content/en/blog/2023/from-zero-to-subproject-lead.md

```diff
@@ -0,0 +1,77 @@
+interesting. Likewise, continued participation is what keeps the community thriving, so that
+interest will drive you to have more of an impact. 
+
+The following is also essential - it’s ok to show up knowing nothing! I remember showing up knowing
+very little about Kubernetes or how the community itself worked. And while I know more about how the
+community functions today, I am still learning all the time about it and the project. Fortunately,
+the community is full of friendly people who want to help you learn. Learning as you go is expected
+and celebrated. When you raise your hand to do something, even if you know nothing, people will
+cheer and help you along the way. 
+
+This method was my exact story. It was my first or second meeting with [SIG
+Security](https://github.com/kubernetes/community/tree/master/sig-security), and [Pushkar
+Joglekar](https://github.com/PushkarJ) mentioned that he needed a lead for a subproject he was
+creating after having done a security assessment of CAPI. Everyone was so friendly in the meeting
+that I thought - hey, why not try it out? And since then, I have received so much support and
+encouragement from my co-leads who are delighted to have me, _especially_ because I am a beginner:
+new participation is what keeps the community healthy.
+
+## Always learning
+
+My participation has also been a great learning experience on several fronts. First, I have been
+exposed to techniques for how to build community consensus. It’s simple stuff: show up at other SIG
+or working group meetings, share your ideas or where you are looking for help, find people who are
+interested and have the knowledge to help, build an action plan together, do it, and share as you
+execute. But the other thing that I’m learning is that building this consensus and executing it in a
+transparent, inviting way simply takes time. 
+
+I also have to be patient with myself and remember that I am learning as I go. The [Kubernetes git
+repo](https://github.com/kubernetes/kubernetes) can be daunting to navigate. Knowing the next best
+step isn’t always obvious. But this is where my third learning curve - how to engage the community
+to get what I need - comes into play. It turns out that asking questions in the [Kubernetes Slack
+workspace](https://slack.k8s.io/) and bringing my topics to the SIG Security meetings when I need
+help is an amazing way to get what I need! Again, simple stuff, but until you do it, it’s not always
+obvious.
+
+## Why you - a beginner - are important to the project
+
+In many ways, beginners are the most important part of the community. To put a finer point on it:
+asking for, receiving, and then giving help is a very relevant part of how the community grows and
+flourishes. When we take on and then pass on knowledge, we ensure that the community grows enough to
+keep supporting the needs of the people who rely on the project, whatever it is. You have
+superpowers as a beginner! 
```

Comment:
If you want to, emphasize that beginners can see problems (eg a gap in the docs) that experienced contributors find hard to see. It really is a superpower.

Reply:
nit on suggestion: “truly”